spec/fizz_buzz_spec.rb

```diff
@@ -1 +1,48 @@
 require "fizz_buzz"
+
+describe FizzBuzz do
+
+  describe ".divisible_by" do
+
+    context "given number, returns number" do
```

Comment:
Good use of context here.  It clearly tells me the following tests will test the non-Fizz/Buzz/FizzBuzz case